### DISCUSSION TITLE
Lora models not loading when they are outside installation folder

### DISCUSSION THREAD

**Discussion by elen07zz** [vladmandic/sdnext#897] [General]
Hello, no lora is being loaded if it is not the default location.
I tried to change the default location to the one I use in automatic1111
but it does not show any, but it seems to detect the subfolders inside, but without any content.

/home/user/stable-diffusion-webui/models/Lora

**Comment by jasparagus** [comment]
I'm having this issue as well. I'm additionally seeing the same behavior with Lycoris models not showing up.

Mine are likewise in folders outside the vlad install location, since they are shared between vlad, auto, and comfyUI.

Oddly, my textual inversions are working just fine (they're also in an outside directory).

**Comment by fabiobtex** [comment]
Same with ESRGAN upscalers

**Comment by Iridium85** [comment]
I have all model files outside the default directory. Checkpoints and textual inversions work as before, but no Lora models are found. I don't have any hypernetworks or LyCORIS so I can't tell if they are affected. I tried changing the path for Loras to something else and back again in the System Paths settings, but they still don't show up.

**Comment by vladmandic** [comment]
converting this discussion into issue #898
